### DISCUSSION TITLE
Nutzen von CUxd Devices

### DISCUSSION THREAD

**Discussion by martin1673** [SukramJ/aiohomematic#1010] [Q&A] [answered]
Hallo,

ich habe gerade den Homeassistant mit der Homematic (Rasperrymatic) gekoppelt.
Den Homeassistent habe ich heute neu instlliert und über HACS die anbindung an die Homematic.
Jetzt habe ich alle HM und HM IP geräte finden können, hae aer Historisch noch einige FS20 eräte die ich über CUxD mit nem eigenen Cul betreibe.
Diese bekomme ich derzeit nicht angezeigt.
Gibt es hier schon einen Trick/Weg diese mit einzubinden?

Es sind einige Steckdosen und Schalter die noch wunderbar Funktionieren, daher werden die nicht priorisiert ausgetauscht.

**Comment by danielperna84** [comment, marked as answer]
CUxD wird nicht unterstützt. Die Integration basiert auf dem `XML-RPC` Protokoll, welches CUxD jedoch nicht bereitstellt. Hier habe ich schon was zur Problematik geschrieben: https://github.com/danielperna84/pydevccu/issues/34

**Comment by martin1673** [comment]
Ok.. danke für die schnelle rückmedung
